packages/amplify-frontend-flutter/CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# Change Log
```

Comment:
How is this new? Did we rename the package or coming from a wrong merge/rebase? PRs should not have changelog files.

Reply:
Not an issue in PR.